nlp/admin/web/package.json

```diff
@@ -27,7 +27,7 @@
     "@nebular/theme": "10.0.0",
     "@ng-bootstrap/ng-bootstrap": "^13.1.1",
     "ang-jsoneditor": "^1.10.5",
-    "bootstrap": "^5.2.0",
+    "bootstrap": "^4.4.1",
```

Comment:
why this version downgrade ?

Reply:
Je ne sais pas pourquoi Brandon avait upgradé bootstrap uniquement sur Nlp, sans doute une erreur, toujours est-il que Nlp ne compilait plus et que l'upgrade ne peut pas se faire partiellement entre Nlp et Bot sinon la compatibilité entre les deux parties de l'app est compromise

Reply:
Donc ici downgrade en rapport avec le bot/admin/web mais si on monte à 5.2.0 les deux, ça devrait passer ?

Reply:
non, c'est une version avec breaking changes. Et puis on a aucun usage des nouveautés introduites